tap_mambu/sync.py

```diff
@@ -805,7 +805,8 @@ def sync(client, config, catalog, state):
             },
             'bookmark_field': 'occurred_at',
             'bookmark_type': 'datetime'
-        }
+        },
+        'interest_accrual_breakdown': {}
```

Comment:
Otherwise, if we can't get it running in tap-tester with records. I'd like some kind of confirmation that sending the data to Stitch succeeds (e.g., through an Import API connection and some tap logs while running against `target-stitch`). There can be subtle things that cause validation errors, so I like to have this before adding anything new.

Reply:
Sure, I'll talk to Alex tomorrow about this, as we finished for today.